### ISSUE TITLE
buildah run cannot parse `--runtime-flag` 

### ISSUE THREAD

**Issue by gouyang** [projectatomic/buildah#428]
The manpage of buildah-run says:
```
 --runtime-flag flag

       Adds  global  flags  for the container runtime. To list the supported flags, please consult manpages of your selected container runtime
       (runc is the default runtime, the manpage to consult is runc(8))
```

Tried it with the runc global option, it does not work and the message isn't helpful.
```
# buildah run busybox-working-container --runtime-flag log
No help topic for 'log'
# buildah run busybox-working-container --runtime-flag debug
No help topic for 'debug'

```
The problem also happens in `buildah bud` etc. I'm not sure whether I understand it right, please comment.

**Comment by TomSweeneyRedHat** [comment]
@gouyang  Thanks for the issue.   It looks like the runtime-flag is not working as intended.  This should work (at least according to the manpage and it's not.  @umohnani8 got time to look at this one?

```
buildah run alpine-working-container --runtime-flag --no-new-keyring -- ps -auxw
option --runtime-flag requires a value
```

Looks like some kind of cli parameter handling issue.

**Comment by umohnani8** [comment]
Yup, I can work on this.

**Comment by rhatdan** [comment]
I believe this might be confusing the CLI Interpreter.

**Comment by umohnani8** [comment]
Fixed in #431
